api/nodes/serializers.py

```diff
@@ -398,6 +399,9 @@ def get_absolute_url(self, obj):
     def get_logs_count(self, obj):
         return obj.logs.count()
 
+    def get_fork_count(self, obj):
+        return obj.forks.count()
```

Comment:
and implemented this to exclude registration and deleted forks: https://github.com/CenterForOpenScience/osf.io/pull/8370/files#diff-7a6d21a50624153d15e63a5f643401c7R482